packages/react/src/internal/field-label/FieldLabel.tsx

```diff
@@ -6,7 +6,9 @@ import { Tooltip } from '../../components/tooltip';
 
 type FieldLabelProps = {
   hidden?: boolean;
+  id?: string;
   inputId: string;
+  isAriaLabelledBy?: boolean;
```

Comment:
I was thinking would it be possible to just provide one value but then again there might be other reasons why the label has to have an id 🤔

Reply:
Yes, I feel you but that was my reasoning too.